drivers/mpu9250/MPU9250.cpp

```diff
@@ -408,31 +321,27 @@ void MPU9250::_measure()
 	}
 
 	if (int_status & BITS_INT_STATUS_FIFO_OVERFLOW) {
-		DF_LOG_ERR("overflow");
 		reset_fifo();
 
 		m_synchronize.lock();
-		++m_sensor_data.fifo_overflow_counter;
+		if ((++m_sensor_data.fifo_overflow_counter % 1000) == 0) {
+			DF_LOG_ERR("FIFO overflow");
```

Comment:
Nice! However, I was hoping that such an overflow gets resolved straightaway. If we really end up stuck in this overflow mode for >1000 samples, we're in trouble...